src/notifyGeneralChannel.ts

```diff
@@ -6,6 +6,9 @@ import { log } from './logger';
 const { GENERAL_CHANNEL } = config;
 
 export const notifyGeneralChannel = async (channel: DMChannel | GuildChannel) => {
+    // Note: When the bot sends DMS to members, this is triggered. So we ignore the notification
+    if (channel instanceof DMChannel) return
```

Comment:
Good spot 👍 

I think our style is more like this...

```suggestion
    if (channel instanceof DMChannel) {
       return;
    }
```